packages/block-editor/src/store/actions.js

```diff
@@ -1256,12 +1256,27 @@ export function replaceInnerBlocks(
  *
  * @return {Object} Action object.
  */
-export function toggleBlockMode( clientId ) {
-	return {
-		type: 'TOGGLE_BLOCK_MODE',
-		clientId,
+export const toggleBlockMode =
+	( clientId ) =>
+	( { select, dispatch } ) => {
+		if ( ! clientId ) {
+			return;
+		}
+		const block = select.getBlocksByClientId( clientId )[ 0 ];
```

Comment:
Minor nitpick to get rid of those unsightly `[ 0 ]`s 😛

```suggestion
		const [ block ] = select.getBlocksByClientId( clientId );
```